lib/modules/neverEndingReddit.js

```diff
@@ -493,8 +493,8 @@ modules['neverEndingReddit'] = {
 							modules['neverEndingReddit'].NERFail(noresultsfound);
 						}
 
-						var e = document.createEvent('Events');
-						e.initEvent('neverEndingLoad', true, true);
+						// fire an event so other modules can respond to page loads.
+						var e = new Event('neverEndingLoad');
```

Comment:
Sorry for the lack of comment. This event is for the benefit of other extensions, not RES internal modules.